README.md

```diff
@@ -21,31 +21,158 @@ npm install
 ```
 npm start
 ```
-The Admin App is now served on [localhost](http://localhost:3000). When you first open the URL, you are prompted to configure the Sitefinity CMS instance you are working with. In the URL field, enter the instance details and then save the configuration. You can later change the configuration by navigating to [config] (http://localhost:3000/config). 
-
-Once you setup the Sitefinity CMS instance, the server becomes in watch mode and automatically recompiles and serves any newly created files.
 
 3. When you are done developing execute the following command:
 ```
 npm run build
 ```
 As a result, a JavaScript file (**extensions.bundle.js**) is generated in the **dist** folder.
 
-4. Register your extensions with the Admin App by upload this file to the **extensions.bundle.js** file in the **root** of the Sitefintiy CMS web application and then refresh your Sitefinity CMS page.
+4. Register your extensions with the Admin App by uploading this file to the **extensions.bundle.js** file in the **root** of the Sitefintiy CMS web application and then refresh your Sitefinity CMS page.
```

Comment:
- Currently, the file name produced by the build command is sample.extensions.bundle.js. We might include the information that they can rename it to something more meaningful like unit1.extensions.bundle.js and the AdminApp will import anything named *.extensions.bundle.js

- @Gebov did we moved the extensions back to the root?

Reply:
no they are in the adminapp folder

Reply:
lets add the restart